### PR TITLE
Fixes #5242

### LINKED ISSUE #5242: This slutty recycling machine just ate my materials
Added 70 glass and 100 plasma sheets.

They disappeared but didn't show up on the screen and they are gooone.  Recyk - PRO THIEF 

unique_nettle_monitor_iii_2015-07-04_22-18-00

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

**Comment by Probe1** [comment]
thank you based PJB

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -16,7 +16,15 @@ Mineral Sheets
...
 /obj/item/stack/sheet/mineral
-	w_type=RECYK_METAL
+	w_type = RECYK_METAL
+	var/recyck_mat
+
+/obj/item/stack/sheet/mineral/recycle(var/datum/materials/rec)
+	if(!recyck_mat)
+		return 0
+
+	rec.addAmount(recyck_mat, 1 * amount)
```
Comment (ComicIronic) [inline code/game/objects/items/stacks/sheets/mineral.dm]: A yes, 1 * the number

Comment (PJB3005) [inline code/game/objects/items/stacks/sheets/mineral.dm]: derp, it was like that in the oldcode too and my brain derped

**Comment by PJB3005** [comment]
>pomf merged it before I fixed the 1 *

**Comment by ComicIronic** [comment]
It took you five hours?

**Comment by PJB3005** [comment]
five hours of playing terraria :^^^^^^^)

**Comment by Probe1** [comment]
no, leave the 1.  its good to preserve it for future coders to remember why oldcode is bad.

**Comment by PJB3005** [comment]
henk

**Comment by Probe1** [comment]
yeah but we love you now for fixing that fucking bug.

**Pull request merged**